index.html

```diff
@@ -45,11 +45,25 @@ <h1>Go for it!</h1>
                     0
                 </td>
             </tr>
+
+            <tr>
+                <td>
+                    Computer chips
+                </td>
+                <td id="chips-amount">
+                    0
+                </td>
+                <td id="chips-income">
+                    0
+                </td>
+            </tr>
+
         </table>
 
         <div class="btn-group">
-            <button id="iron-btn" class="btn btn-secondary">Buy Miner (5 iron)</button>
-            <button id="fuel-btn" class="btn btn-primary">Buy Fuel Maker (15 iron)</button>
+            <button id="iron-btn" class="btn btn-secondary">Buy Miner (5 iron, 1 chip)</button>
+            <button id="fuel-btn" class="btn btn-primary">Buy Fuel Maker (15 iron, 2 chips)</button>
+            <button id="chips-btn" class="btn btn-secondary">Repair chip fabricator (150 iron)</button>
```

Comment:
Before going any further it would be a good idea to move prices into code (as _const_s for now) then generate the button text and purchase checks/deductions based on that single source.

Reply:
I was thinking of doing this yesterday but I wanted to make a dictionary or a function where you pass in what you are trying to buy and it returns the price of each resource but couldn't quite decide/work out how to do it.